components/AirlineList/index.js

```diff
@@ -0,0 +1,54 @@
+import {
+  calculateVolume,
+  calculateVolumeCombined,
+  convertDimension,
+} from "@/utils";
+
+export default function AirlineList({ sortedAirlines, unitSystem }) {
+  return (
+    <>
+      {sortedAirlines.map(
+        ({ id, name, personalItem, cabinBag, freeCabinBag }) => {
+          return (
+            <div key={id} style={{ border: "1px solid black", padding: 10 }}>
```

Comment:
Please use styled components instead of inline styles

Reply:
> Please use styled components instead of inline styles

Can I use this shortcut in the meantime until I come to the task of styling? I will use StyledComponents at the time, I promise. Just wanted to keep the snippet from Dominik because it made things easier to read in the list.

Reply:
✅